src/Symfony/Component/HttpFoundation/File/Exception/CantWriteFileException.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpFoundation\File\Exception;
+
+/**
+ * Thrown when an error UPLOAD_ERR_CANT_WRITE occurred with UploadedFile.
+ *
+ * @author Florent Mata <florentmata@gmail.com>
+ */
+class CantWriteFileException extends FileException
```

Comment:
I would nose Cant here but Cannot instead.

Reply:
I wanted to follow the PHP const `UPLOAD_ERR_CANT_WRITE` to keep consistency, as others exceptions added in this PR. Are you sure of this change ?

Reply:
@fmata can you please do the change? (and rebase meanwhile to fix the small conflict)

Reply:
@nicolas-grekas rebased & updated